lib/node/runner/isaac_state_manager.go

```diff
@@ -229,6 +231,9 @@ func (sm *ISAACStateManager) Start() {
 				}
 				sm.setState(state)
 				sm.transitSignal(state)
+				if state.BallotState == ballot.StateALLCONFIRM {
+					begin = metrics.Consensus.SetBlockIntervalSeconds(begin)
```

Comment:
I think `INIT` to `INIT` is better than `ALLCONFIRM` to `ALLCONFIRM` for calculating block interval :)

Reply:
@Charleslee522 I changed it. thanks.

Reply:
👍 
Checking `ballot.StateINIT` is already https://github.com/bosnet/sebak/pull/708/commits/cba73a95c036c07a0b6a253379657f382508f43f#diff-3c264302e46a605f9bcfe3b107a91703R225.
I think it can be called above :)